.github/cloudbuild/pos-pr-open-to-main.yaml

```diff
@@ -101,10 +101,43 @@ steps:
   dir: 'point-of-sale-app'
   args: [
       'bash',
-      '/workspace/deploy.sh'
+      '/workspace/deploy.sh',
   ]
   # waitFor: commented out, so this step waits for all previous steps
 
+  ###########################################################
```

Comment:
This change was already reviewed and merged to main in: https://github.com/GoogleCloudPlatform/point-of-sale/pull/101. Thus no need to review it here. It is showing up here because we are not merging to main, but merging to a new branch called `v2-sql` until all the SQL specific stuff are through!